tests/components/orderentry/SelectOrderType.test.jsx

```diff
@@ -21,14 +21,14 @@ describe('Component: orderentry: SelectOrderType', () => {
   });
   it('should render three buttons', () => {
     const component = getComponent();
-    expect(component.find('button').length).toBe(2);
+    expect(component.find('.order-type-option').length).toBe(2);
```

Comment:
This test contradicts the description of rendering three buttons. I also think, to test for the number of elements rendered is a bit redundant as these change all the time and tests would have to be edited every time a button is added there or else the tests would fail. But it was there initially so maybe re-write it or correct me if I am mistaken.

Reply:
a typo, I'll look into this..